dev-qt/qtwebengine/patches/qtwebengine-5.12.3.patchset

```diff
@@ -0,0 +1,4832 @@
+Date: Sun Sept 9 2018 08:10:16 +0000 
+From: Ken Mays <kmays2000@gmail.com>
+Subject: Qt WebEngine 5.11.2 patchset
+
+From: Jan Steffens <heftig@archlinux.org>
+Date: 2017-09-19 23:30:08 -0700
+Subject: This is required for QtWebEngine to show the patent-free LCD rendering. Without this patch, only grayscale rendering is used.
```

Comment:
Is this relevant?

Reply:
Still applies to QTWebEngine 5.12.3. Haiku/AGG - I'd leave it for now.

Reply:
I already asked you more than once in your earlier qtwebengine PRs why is this required, but  so fa got no explanationr, so i assume this is just something what you want to have, which is nice, but this doesn't makes the Haiku port easier (actually it **can** even create troubles), and technically this have nothing to do with the Haiku port, so i still say **currently** it is unnecessary. If it builds fine, then can we start to experiment with 3rdparty patches, and enable more features.